openapi-diff/src/modeler/AutoRest.Swagger/Model/Response.cs

```diff
@@ -46,6 +46,7 @@ public class OperationResponse : SwaggerBase<OperationResponse>
             var headers = Headers != null ? Headers : new Dictionary<string, Header>();
             var priorHeaders = priorResponse.Headers != null ? priorResponse.Headers : new Dictionary<string, Header>();
 
+            context.PushProperty("headers");
             foreach (var header in headers)
```

Comment:
remind me whats the purpose of this context ?

Reply:
to keep path/reference to the object